chipsec/modules/tools/vmm/xen/hypercallfuzz.py

```diff
@@ -1,76 +1,70 @@
-#CHIPSEC: Platform Security Assessment Framework
-#Copyright (c) 2010-2021, Intel Corporation
-#
-#This program is free software; you can redistribute it and/or
-#modify it under the terms of the GNU General Public License
-#as published by the Free Software Foundation; Version 2.
-#
-#This program is distributed in the hope that it will be useful,
-#but WITHOUT ANY WARRANTY; without even the implied warranty of
-#MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#GNU General Public License for more details.
-#
-#You should have received a copy of the GNU General Public License
-#along with this program; if not, write to the Free Software
-#Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-#
-#Contact information:
-#chipsec@intel.com
-#
-
+# CHIPSEC: Platform Security Assessment Framework
+# Copyright (c) 2010-2021, Intel Corporation
+# 
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; Version 2.
+# 
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+# 
+# Contact information:
+# chipsec@intel.com
+# 
 
 
 """
 Xen hypercall fuzzer
 
 Usage:
-  ``chipsec_main.py -i -m tools.vmm.xen.hypercallfuzz \``
-  ``-a <mode>[,<vector>,<iterations>] -l log.txt``
+    ``chipsec_main.py -i -m tools.vmm.xen.hypercallfuzz -a <mode>[,<vector>,<iterations>]``
 
-    - ``mode``				fuzzing mode
+    - ``mode``                       : fuzzing mode
 
-        * ``= help``			prints this help
-        * ``= info``			hypervisor information
-        * ``= fuzzing``			fuzzing specified hypercall
-        * ``= fuzzing-all``		fuzzing all hypercalls
-        * ``= fuzzing-all-randomly``	fuzzing random hypercalls
-    - ``vector``			code or name of a hypercall to be fuzzed (use info)
-    - ``iterations``			number of fuzzing iterations
+        * ``= help``                 : Prints this help
+        * ``= info``                 : Hypervisor information
+        * ``= fuzzing``              : Fuzzing specified hypercall
+        * ``= fuzzing-all``          : Fuzzing all hypercalls
+        * ``= fuzzing-all-randomly`` : Fuzzing random hypercalls
+    - ``<vector>``                   : Code or name of a hypercall to be fuzzed (use info)
+    - ``<iterations>``               : Number of fuzzing iterations
 
 Examples:
+    >>> chipsec_main.py -i -m tools.vmm.xen.hypercallfuzz -a sched_op,10 -l log.txt
```

Comment:
None of the `mode`s above are reflected in the arguments in the examples...? Might also make sense to remove the `= ` in the modes?